service/matching/task_queue_partition_manager.go

```diff
@@ -526,25 +526,51 @@ func (pm *taskQueuePartitionManagerImpl) Describe(
 	buildIds map[string]bool,
 	includeAllActive, reportStats, reportPollers, internalTaskQueueStatus bool) (*matchingservice.DescribeTaskQueuePartitionResponse, error) {
 	pm.versionedQueuesLock.RLock()
+
```

Comment:
would love to hop on a call to discuss the changes to this file -- I think that will be the fastest way for me to review this part

Reply:
change request: comment at top of fn and in the two build-id cases about how you're accepting version id strings as build ids and returning them as build ids